crates/dreammaker/src/parser.rs

```diff
@@ -944,6 +950,8 @@ impl<'ctx, 'an, 'inp> Parser<'ctx, 'an, 'inp> {
             } else {
                 last_part.push_str("[]");
             }
+        } else if self.exact(Punct(EmptyQuote))?.is_some() {
```

Comment:
A less shocking change might be to look for an empty `Token::String` here.

Reply:
handled, I think?